includes/sanitizers/class-amp-gallery-block-sanitizer.php

```diff
@@ -140,8 +140,41 @@ public function sanitize() {
 					'layout' => 'responsive',
 				]
 			);
+
 			foreach ( $images as $image ) {
-				$amp_carousel->appendChild( $image );
+				$div = AMP_DOM_Utils::create_node(
```

Comment:
This used to simply append the image to the `<amp-carousel>`. But now it creates a `<div class="slide">` wrapper for the image, and possibly the caption.

It looks like that's needed in order to display a caption.